README.md

```diff
@@ -68,51 +68,49 @@ app = Starlite(route_handlers=[hello_world])
 - [Life Cycle Hooks](#request-life-cycle-hooks)
 - [Route Guards based Authorization](#route-guards)
 - Layered Parameter declaration
-- SQLAlchemy Support (via plugin)
-- Piccolo ORM Support (via plugin)
-- Tortoise ORM Support (via plugin)
+- SQLAlchemy plugin support
+- Piccolo ORM plugin support
+- Tortoise ORM plugin support
 - Extended testing support
 - [Automatic API documentation with](#redoc-swagger-ui-and-stoplight-elements-api-documentation):
   - [Redoc](https://github.com/Redocly/redoc)
   - [Stoplight Elements](https://github.com/stoplightio/elements)
   - [Swagger-UI](https://swagger.io/tools/swagger-ui/)
 - Support for dataclasses and `TypedDict`
 - [Trio](https://trio.readthedocs.io/en/stable/) support (built-in, via [AnyIO](https://anyio.readthedocs.io/))
-- Ultra-fast json serialization and deserialization using [msgspec](https://github.com/jcrist/msgspec)
+- Ultra-fast JSON serialization and deserialization using [msgspec](https://github.com/jcrist/msgspec)
 
-## Example Applications
+## Example applications
 
 - [starlite-pg-redis-docker](https://github.com/starlite-api/starlite-pg-redis-docker): In addition to Starlite, this
   demonstrates a pattern of application modularity, SQLAlchemy 2.0 ORM, Redis cache connectivity, and more. Like all
   Starlite projects, this application is open to contributions, big and small.
 - [starlite-hello-world](https://github.com/starlite-api/starlite-hello-world): A bare-minimum application setup. Great
-  for testing and POC work.
+  for testing and Proof-Of-Concept (POC) work.
```

Comment:
I'd swap, POC and then in parans the explication

Reply:
So something like this - POC (Proof-Of-Concept)? Yeah that works too I guess & this same style has to be maintained across the entirety of the docs then.